src/test/s3/util/fileViewerManager.test.ts

```diff
@@ -152,4 +165,13 @@ describe('FileViewerManager', function () {
             assert.deepStrictEqual(await fileViewerManager.getFromTemp(testNode), undefined)
         })
     })
+
+    describe('on save', function () {
+        beforeEach(async function () {})
+
+        it('uploads back to s3, closes and reopens in edit mode', function () {
```

Comment:
Can this test be finished with the current PR or is it dependent on another?

Reply:
next PR